package/yast2.changes

```diff
@@ -1,4 +1,12 @@
 -------------------------------------------------------------------
+Tue Mar 28 10:19:44 WEST 2017 - knut.anderssen@suse.com
+
+- backported
+  - SlideShow: Escape plain text release notes being shown properly
+    in RichText (bsc#1028721).
+- 3.1.215
```

Comment:
@jreidinger As I understood the agreement with QA, we should merge code from CaaSP into SP2 as often as possible. It means some MU will contain code that is not there to fix a real SP2 bug, but only to keep SP2 and CaaSP in sync.

That's expected and accepted by QA, as I understood, since it's less harmful that having a **huge** MU for SP2 released at the CaaSP release day and containing everything.

@kobliha, @jsrain Please correct me if I'm wrong.